web/src/lightbox.js

```diff
@@ -177,18 +177,32 @@ export function clear_for_testing() {
 export function render_lightbox_list_images(preview_source) {
```

Comment:
I do realize this commit require name changes but to me it looks like this library need to rewritten in a better way. So, I am trying to do that next.